### PR TITLE
Polygon

### PR DESCRIPTION
**Pull request by Plygon**

### CODE REVIEW

File: class/Joint.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ export class Joint {
     constructor(a, b, parent) {
         this.a = a;
         this.b = b;
-        this.bike = parent;
```
Comment (tomiy) [inline class/Joint.js:7]: Is this intentional? @Plygon

Comment (tomiy) [reply class/Joint.js:7]: Pretty sure the parent is always the bike

Comment (Plygon) [reply class/Joint.js:7]: It is intentional.
Joints are also between limbs of the rider when he falls off

Comment (tomiy) [reply class/Joint.js:7]: Makes sense. Thanks!

**Pull request merged**